vercel.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
This seems very global in nature...  will it have unintended side effects?

Reply:
i haven't experienced any unintended side effects using this. i've changed the configuration to only cache those json files still